index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>MY Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <header>
+      <div class="nav">
+        <nav>
+          <div class="logo">
+            <img src="images/image_geometry_1.svg" alt="logo" />
+            <div class="menu-icon">
+              <img src="images/new-menu-icon.png" alt="menu-icon" />
+            </div>
+          </div>
+        </nav>
+      </div>
+    </header>
+    <section class="home-section">
+      <div class="head">Ashwin <br />Carvalho</div>
+      <div class="paragraph">
+        Hello! I am a software developer! I can <br />
+        help you build a product, feature or <br />
+        website. Taka a look of my works.<br />
+        If you like what you see and have a<br />
+        project you need coded, don`t hesitate<br />
+        and contact me.
+      </div>
+      <button class="button">Start collaboration</button>
+      <div class="project-head">Project</div>
+
+      <!-- com image  -->
+
+      <div class="computer-img"></div>
+      <div class="computer-img-content"></div>
+      <div class="com-head">Project name goes here</div>
+      <div class="com-border-text">
+        <ul>
+          <li>HTML/CSS</li>
+          <li>Ruby on Rails</li>
+          <li>JavaScript</li>
+        </ul>
+      </div>
+      <div class="see-project">
+        See this project
+        <div class="img-right-arrow">
+          <img src="/images/Right-Arrow.png" alt="Right Arrow" />
+        </div>
+      </div>
+
+      <!-- com image 1 -->
+
+      <div class="computer-img-1"></div>
+      <div class="computer-img-content-1"></div>
+      <div class="com-head-1">Project name goes here</div>
+      <div class="com-border-text-1"></div>
+      <ul>
+        <li>HTML/CSS</li>
+        <li>Ruby on Rails</li>
+        <li>JavaScript</li>
+      </ul>
+
+      <div class="see-project-1">
+        See this project
+        <div class="img-right-arrow-1">
+          <img src="/images/Right-Arrow.png" alt="Right Arrow" />
```

Comment:
- [ ] Please check all image paths again. You are trying to load images from `images` folder, but there is no `images` folder in [Portfolio-new-full](https://github.com/AshwinCarvalho999/Portfolio-new-full) branch. Please be sure all images are loaded correctly and are visible. 
- [ ] You should also use relative paths for the image path in order to make them visible on all systems. On some operating systems or servers, images may not get loaded because of this issue. Please refactor the image path with relative paths. 
```suggestion
          <img src="images/Right-Arrow.png" alt="Right Arrow" />
```

https://github.com/AshwinCarvalho999/Portfolio-new/pull/2#discussion_r917264815

> * [ ]  Kindly make sure all your images are displaying.